### ISSUE TITLE
cloudinary-react/samples/photo_album/ does not work

### ISSUE THREAD

**Issue by 2stash** [cloudinary/cloudinary-react#194]
## Bug report for Cloudinary React SDK
Before proceeding, please update to latest version and test if the issue persists

## Describe the bug in a sentence or two.
cloudinary-react/samples/photo_album/ does not work. It returns an error about upload presets. 

## Issue Type (Can be multiple)
[ ] Build - Can’t install or import the SDK
[ ] Babel - Babel errors or cross browser issues
[ ] UI/Performance - Display or performance issues
[x ] Behaviour - Functions aren’t working as expected (Such as generate URL)
[ ] Documentation - Inconsistency between the docs and behaviour
[ ] Incorrect Types - For typescript users who are having problems with our d.ts files
[ ] Other (Specify)

## Steps to reproduce
Clone cloudinary-react/samples/photo_album/. Follow steps in readme. Does not work.

## Error screenshots
… if applicable

## Browsers (if issue relates to UI, else ignore)
[ ] Chrome
[ ] Firefox
[ ] Safari
[ ] Other (Specify)
[ ] All

## Versions and Libraries (fill in the version numbers)
React Cloudinary SDK - 0.0.0
React - 0.0.0
Create-React-App - 0.0.0 / N/A
Babel - 0.0.0
Webpack/Rollup/Other - 0.0.0
Node - 0.0.0
NPM - 0.0.0

## Config Files (Please paste the following files if possible)
Package.json
Webpack.config (if applicable)
Babelrc

## Repository
If possible, please provide a link to a reproducible repository that showcases the problem

**Comment by shirlymanor** [comment]
Can you please share more details about your issue?

**Comment by 2stash** [comment]
Hi. I am looking for an image hosting service to integrate with the backend
of the application I am working on. I found cloudinary and thought I would
give it a try. I tried the github sample project and it doesn't work. See
error below. Per the readme, this shouldn't happen as it specifically
states the presets are set in the client of the sample. So I searched
cloudinary's docs and the stuff I found about presets was adding a bunch of
features like face recognition, which I don't need, nor is it related to
setting up this sample. So I must have missed something completely, or this
service is really complicated to setup. The next thing I found was a 16hour
course offered by cloudinary, which I don't really want to go through, to
figure out how to run a sample project.

There are only three instructors with the sample, so I assume I am doing
one wrong?
1) cloud name changed
2) I enabled unsigned presets. I assume the upload preset UUUUUUUUUUUU
means something.
3) I didn't find a "list" item to uncheck or check.

export default {
    cloud_name: 'discimages',
    upload_preset: 'UUUUUUUU'
};



[image: image.png]


[image: image.png]
